app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    cmd, source_path, destination_path = parts
```

Comment:
create a separate variable for `os.getcwd()` to avoid calling the function one more time